.build/tar/run.sh

```diff
@@ -0,0 +1,3 @@
+echo "Thank you for using OpenTelemetry Collector and Google Cloud! You are currently running the binary with the default configuration file for Google Cloud Monitoring. If you want to add, edit, or remove receivers, processors, or exporters, please directly edit the file 'config-example.yaml'"
```

Comment:
Not sure if we need this comment again? It's already in the README

Reply:
My goal here is to have a full coverage of comments like this. However, if you find this unnecessary, I can just remove the run.sh completely because there would only be one command here.

Reply:
Yea I dunno about this. I guess I don't mind too much either way, I just have a general preference for less stuff. In particular, less stuff that could possibly get out of sync (e.g. if someone built a tarball with a different config)

Also btw I think this is currently broken since you renamed the config file 😄

Reply:
Ok, since there is only one command in here, I am not very strong in keeping this as well. For simplicity, I have just removed this file. If we ever see the need for adding this back, we can always do that.